### PR TITLE
MAT-2279 Test Execution with Episode of Care

### PR DESCRIPTION
**Pull request by adongare**
MAT-2279 Test Execution with Episode of Care

Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/helpers/calculator_helpers.js [hunk 1]
```diff
@@ -268,17 +268,17 @@ module.exports = class CalculatorHelpers {
             dataElements.forEach((dataElement) => {
               if (dataElement.id != null) {
                 // if an episode has already been created set the result for the population to 1
-                if (episodeResults[dataElement.id]) {
-                  episodeResults[dataElement.id][popCode] = 1;
+                if (episodeResults[dataElement.id.value]) {
```
Comment (adongare) [inline lib/helpers/calculator_helpers.js:271]: @serhii-ilin can this be adjusted in typescript models? we can discuss if I'm not clear here

**Review by jkotanchik-SB — APPROVED** [review]

File: lib/helpers/calculator_helpers.js (hunk 1 above)
Comment (serhii-ilin) [reply lib/helpers/calculator_helpers.js:271]: Not sure what you mean..

**Review by serhii-ilin — APPROVED** [review]

File: dist/browser.js [hunk 2]
```diff
@@ -276,17 +276,17 @@ module.exports = class CalculatorHelpers {
             dataElements.forEach((dataElement) => {
               if (dataElement.id != null) {
                 // if an episode has already been created set the result for the population to 1
-                if (episodeResults[dataElement.id]) {
-                  episodeResults[dataElement.id][popCode] = 1;
+                if (episodeResults[dataElement.id.value]) {
```
Comment (serhii-ilin) [inline dist/browser.js:279]: does it make sense to introduce a local id variable? the same expression is used a couple of times

Comment (adongare) [reply dist/browser.js:279]: It's just accessing the object property. doesn't seem to be expensive?

File: lib/helpers/calculator_helpers.js (hunk 1 above)
Comment (adongare) [reply lib/helpers/calculator_helpers.js:271]: maybe we'll discuss it in my next PR. I'll merge this so that Mike would be able to create patient for further testing
